addons/website/views/website_views.xml

```diff
@@ -555,12 +555,10 @@
             <field name="model">ir.module.module</field>
             <field name="mode">primary</field>
             <field name="arch" type="xml">
-                <form create="false" edit="false" delete="0" js_class="theme_preview_form">
+                <form create="false" edit="false" delete="0" js_class="theme_preview_form" class="o_view_form_theme_preview">
```

Comment:
Should the class be added by the JS, seems strange to allow `js_class="theme_preview_form"` without the class no?